website/src/content/docs/docs/howtos/Generate client libraries/08types.mdx

```diff
@@ -1960,7 +1916,9 @@ utf8JsonWriter.WriteStringValue(Prop.ToString());
 <ClientTabItem lang="typescript" >
 
 ```typescript
-TODO;
+export interface Test {
+  prop: Date;
+}
```

Comment:
Please add serializer/deserializer showcase for this because different data encoding will result in different serializer/deserializer